### ISSUE TITLE
Limit Talk "user search" based on member group

### ISSUE THREAD

**Issue by morten44** [nextcloud/spreed#5442]
Hi
Thanks for a fantastic product

We have a server with some small business and different schools
When using talk people can search across groups
So a primary school student can call up admin group in a business on same server.

Question
Is there a way that people member of a group called "PrimarySchool", only can search people and groups within this groups?

Right now i can only find a global setting to enable "talk" for groups, so we had to limit the use of "talk" to one group (small business) so the schools / classes can not use it

Is it correct that this is not possible?
Is it on a plan to implement this control?

Thanks again
Kind Regards
Morten

**Comment by nickvergessen** [comment]
This is controlled by the share settings on `index.php/settings/admin/sharing`

**Comment by morten44** [comment]
Hi Nickvergessen
Thanks for your reply.
I went to that page and saw some sharing settings.
I cant find any specific for "talk"
The one I have marked says : Restrict users to only share with users in their group

Questions: So would this restrict "talk" to only search talk members in same group"
If yes i guess folders of files can also only be shared with members of out groups
Is this correct?
Kind Regards
Morten

![Screen Shot 03-31-21 at 11 59 AM](https://user-images.githubusercontent.com/40831248/113134358-99139580-9218-11eb-9e2e-ba0920372d4a.PNG)
